integration/container/create_test.go

```diff
@@ -528,3 +528,45 @@ func TestCreatePlatformSpecificImageNoPlatform(t *testing.T) {
 	)
 	assert.NilError(t, err)
 }
+
+func TestCreateWorkingDirForUser(t *testing.T) {
```

Comment:
Looks like this is failing on Windows (maybe it doesn't expect the UID/GID when creating; wondering if this was an existing bug that _happened_ to be working with `root` (uid/gid `0:0` therefore ignored? 🤔); just an initial hunch


```
=== RUN   TestCreateWorkingDirForUser
    create_test.go:555: assertion failed: error is not nil: Error response from daemon: container bab4875be8b71f227669cd1de1c40034f6bb127d96b860e9468b1a8e0126371e encountered an error during hcs::System::CreateProcess: failure in a Windows system call: The user name or password is incorrect. (0x52e)
--- FAIL: TestCreateWorkingDirForUser (1.60s)
```